Sources/Fuzzilli/Mutators/ExplorationMutator.swift

```diff
@@ -88,6 +89,9 @@ public class ExplorationMutator: Mutator {
                 // exploration, leading to a failed execution. Failed executions are therefore expected, but if the failure rate appears unreasonably high, one could call
                 // maybeLogFailingExecution here and enable verbose mode to investigate.
                 return failure(.instrumentedProgramFailed)
+            } else if case .differential = execution.outcome {
```

Comment:
If we disable differential fuzzing for the instrumentedProgram (see above), just `fatalError` if we see a `.differential` result here instead.